cypress/support/config/settings.js

```diff
@@ -1841,8 +1841,8 @@ module.exports = () => ({
             enabled: false,
           },
           test: {
-            paths: [],
-            enabled: false,
+            paths: ['/indonesia/bbc_indonesian_radio/w172xh26yzb93p7'],
+            enabled: true,
```

Comment:
We're going to have to figure out how to get the most recent episode from the schedule... there's probably something we can do in the index.js file

Reply:
We've agreed to address this in a separate PR - this URL will work for the next 30 days or so, so we have time to figure it out :)